Mathlib/Algebra/Category/ModuleCat/Monoidal/Basic.lean

```diff
@@ -103,6 +104,15 @@ instance instMonoidalCategoryStruct : MonoidalCategoryStruct (ModuleCat.{u} R) w
   leftUnitor := leftUnitor
   rightUnitor := rightUnitor
 
+attribute [moduleCat_simps] ModuleCat.MonoidalCategory.tensorObj
+  ModuleCat.MonoidalCategory.tensorHom ModuleCat.MonoidalCategory.whiskerRight
+  ModuleCat.MonoidalCategory.whiskerLeft ModuleCat.MonoidalCategory.associator
+  ModuleCat.MonoidalCategory.leftUnitor ModuleCat.MonoidalCategory.rightUnitor
+  instMonoidalCategoryStruct_tensorObj instMonoidalCategoryStruct_tensorHom
+  instMonoidalCategoryStruct_whiskerRight instMonoidalCategoryStruct_whiskerLeft
+  instMonoidalCategoryStruct_associator instMonoidalCategoryStruct_leftUnitor
+  instMonoidalCategoryStruct_rightUnitor
```

Comment:
I think I can remove the first half of these lemmas, let me have a go...

Reply:
#24823 I think is the right approach here; in theory `simps` could generate these lemmas too.

Reply:
Fair enough, thanks, I'll try and get that branch compiling